docs/content/docs/guide/authoringprs.md

```diff
@@ -136,6 +141,20 @@ suffix) in the `docs` directory :
       event == "pull_request" && "docs/*.md".pathChanged()
 ```
 
+This example will match any changed file (added, modified, removed or renamed) that was in the `tmp` directory:
```

Comment:
It would be great if you add some clear functionality usage distinction between `pathChanged` and `.matches`

I mean when `pathChanged` can be used and when `.matches` used

Reply:
I think the thought was to deprecate and remove `pathChanged` as the change does everything `pathChanged` did and more. We discussed this with @chmouel a few weeks ago. Has that changed? If not, should we deprecate `pathChanged` in this PR?

Reply:
Yes +1 for deprecating it now and updating docs everywhere to use the latest function instead of `pathChanged`

cc @chmouel